Doc/tutorial/modules.rst

```diff
@@ -191,7 +191,8 @@ named :file:`spam.py` in a list of directories given by the variable
   file is specified).
 * :envvar:`PYTHONPATH` (a list of directory names, with the same syntax as the
   shell variable :envvar:`PATH`).
-* The installation-dependent default.
+* The installation-dependent default (by convention including a
+  ``site-packages`` directory, handled by the :mod:`site` module).
```

Comment:
I feel this is just a noise for tutorial readers.

We don't introduce even standard library (it is introduced in 6.2).  So we shouldn't mention about what is in the "installation-dependent default" in this chapter.  It should be just "some other directories" for the readers here.

If you really want to document site-packages, I think [Python Setup and Usage](https://docs.python.org/3/using/index.html) or [Installing Python Modules](https://docs.python.org/3/installing/index.html) is better place.

Reply:
It's not clear to the reader what "handled" means here.  What would a reader of the tutorial do with the site module?

Reply:
Would "taken care of \[by ...\]" be better, maybe? This is mainly a referral to the first paragraph of the [site module](https://docs.python.org/3/library/site.html) documentation, which says:

> **This module is automatically imported during initialization.**

The tutorial reader, which may be someone having landed here by organic search traffic (i.e. not necessarily a Python novice), may get a hint on what "installation-dependent default\[s\]" are (via "`site-packages`") and where the read more about it (via the link to the `site` module documentation).

Reply:
I guess i don't understand why any reader needs to know what module placed site-packages into the import path?  Just because someone might land on this page through a search doesn't mean we should add every fact to it.  The tutorial should first and foremost be useful to read by beginners.  This addition seems like a confusing addition.